tests/bounties/solidity-bounty/Makefile

```diff
@@ -1,5 +1,5 @@
 ARCH=$(shell uname -m)
-VERSION=0.8.26
+VERSION=0.8.27
```

Comment:
@guidanoli to avoid defining the version 2x in different files (this one and the parent Makefile at tests/bounties) what if we change this setting here by the following:
```
ifndef VERSION
    $(error VERSION is not set. Please provide a VERSION value.)
endif
```

Reply:
Okay for me. :-)